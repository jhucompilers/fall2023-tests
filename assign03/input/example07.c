int main(void) {
  int x;
  int *p;
  volatile int *q;


  p = &x;

  q = p; // this is ok: the base type of the pointer variable being assigned
         // to can have more qualifiers than the pointer type being assigned

  return 0;
}
