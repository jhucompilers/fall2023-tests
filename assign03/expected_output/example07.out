0|main|function|function () returning int
2|x|variable|int
2|p|variable|pointer to int
2|q|variable|pointer to volatile int
